src/RegistryBuilder.php

```diff
@@ -67,7 +67,33 @@ public static function registerSiUnit(Registry $registry, string $name, array $s
 
     protected static function initArea(Registry $registry): void
     {
-        // todo: hectare, etc
+        $registry->register('square_meter', [new UnitPart(1, Dimension::AREA, 2)]);
```

Comment:
```suggestion
        static::registerSiUnit($registry, 'meter^2', ['m2'], 1, Dimension::LENGTH, 2);
```

In general, I think we should register with `^2` instead of `square_`.

Reply:
fine for me !